### PR TITLE
Feature01

### PR DESCRIPTION
**Pull request by suzuki-primestyle**

### CODE REVIEW

**Comment by suzuki-primestyle** [comment]
wants review:

- Is status code valid?

File: test/index.js [hunk 1]
```diff
@@ -18,4 +18,14 @@ describe('Index', () =>{
         }); // chai.request
     }); // it should Get the message "Hello Node!!
   });
+  describe('/GET index expect failing', () => {
+    it('it should get bad status code and fail', (done) => {
+      chai.request(server)
+        .get('/')
+        .end((err, res) => {
+          res.should.have.status(404);
```
Comment (primestyle) [inline test/index.js]: why not do you set the code 301?

**Review by primestyle — CHANGES_REQUESTED** [review]

File: test/index.js (hunk 1 above)
Comment (suzuki-primestyle) [reply test/index.js]: I fixed.

**Review by primestyle — APPROVED** [review]
👍

**Pull request merged**